crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/client/CrimeEvidenceClientTest.java

```diff
@@ -0,0 +1,148 @@
+package uk.gov.justice.laa.crime.crowncourt.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.reactive.function.client.*;
+import reactor.core.publisher.Mono;
+import uk.gov.justice.laa.crime.crowncourt.exception.APIClientException;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
+@ExtendWith(MockitoExtension.class)
+class CrimeEvidenceClientTest {
+
+    public static final String MOCK_URL = "mock-url";
```

Comment:
The below should probably be private

`public static final String MOCK_URL = "mock-url";
`